src/containers/PasswordResetRequestContainer.js

```diff
@@ -0,0 +1,34 @@
+import React, { memo } from 'react'
+import PropTypes from 'prop-types'
+import * as API from '../api/user'
+import ResetRequest from '../components/PasswordResetFlow/ResetRequest'
+
+const PasswordResetRequestContainer = props => {
+  const { history } = props
+
+  const handleReset = async resetInfo => {
```

Comment:
Mixing async/await and promises in handleReset. You could store the response in a variable and use that instead.

Reply:
Good point, fixed this 👍